### PR TITLE
CICD blueprint for CDK

### PR DESCRIPTION
**Pull request by joozero**
## Description
CICD blueprint for CDK 
comprising CodeBuild and CodePipeline

## Motivation and Context
This is common workload in ECS 

## How Has This Been Tested?
- [X] I have tested and validated these changes using one or more of the provided `examples/*` projects
- [X] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

File: cdk/examples/core_infra/lib/core_infra_stack.py
Comment (umairishaq) [inline cdk/examples/core_infra/lib/core_infra_stack.py:1]: I suggest deleting the commented out code.

File: cdk/examples/components/core_infrastructure_construct.py
Comment (umairishaq) [inline cdk/examples/components/core_infrastructure_construct.py:1]: Suggest deleting the commented-out code.

File: cdk/examples/core_infra/lib/core_infra_stack.py
Comment (joozero) [reply cdk/examples/core_infra/lib/core_infra_stack.py:1]: removed

File: cdk/examples/components/core_infrastructure_construct.py
Comment (joozero) [reply cdk/examples/components/core_infrastructure_construct.py:1]: removed

**Review by nvpnathan — APPROVED** [review]

**Pull request merged**